app/i18n/en-us/gen.php

```diff
@@ -0,0 +1,215 @@
+		'category_empty' => 'Empty category',
+		'confirm_action' => 'Are you sure you want to perform this action? It cannot be canceled!',
+		'confirm_action_feed_cat' => 'Are you sure you want to perform this action? You will lose related favorites and user queries. It cannot be canceled!',
+		'feedback' => array(
+			'body_new_articles' => 'There are %%d new articles to read on FreshRSS.',
+			'request_failed' => 'A request has failed, it may have been caused by internet connection problems.',
+			'title_new_articles' => 'FreshRSS: new articles!',
+		),
+		'new_article' => 'There are new articles available, click to refresh the page.',
+		'should_be_activated' => 'JavaScript must be enabled',
+	),
+	'lang' => array(
+		'cz' => 'Čeština',
+		'de' => 'Deutsch',
+		'en' => 'English',
+		'en-us' => 'English (United States)',
+		'es' => 'Español',
+		'fr' => 'Français',
+		'he' => 'עברית',
+		'it' => 'Italiano',
+		'kr' => '한국어',
+		'nl' => 'Nederlands',
+		'oc' => 'Occitan',
+		'pt-br' => 'Português (Brasil)',
+		'ru' => 'Русский',
+		'sk' => 'Slovenčina',
+		'tr' => 'Türkçe',
+		'zh-cn' => '简体中文',
+	),
+	'menu' => array(
+		'about' => 'About',
+		'admin' => 'Administration',
+		'archiving' => 'Archiving',
+		'authentication' => 'Authentication',
+		'check_install' => 'Installation checking',
+		'configuration' => 'Configuration',
+		'display' => 'Display',
+		'extensions' => 'Extensions',
+		'logs' => 'Logs',
+		'queries' => 'User queries',
+		'reading' => 'Reading',
+		'search' => 'Search words or #tags',
+		'sharing' => 'Sharing',
+		'shortcuts' => 'Shortcuts',
+		'stats' => 'Statistics',
+		'system' => 'System configuration',
+		'update' => 'Update',
+		'user_management' => 'Manage users',
+		'user_profile' => 'Profile',
+	),
+	'pagination' => array(
+		'first' => 'First',
+		'last' => 'Last',
+		'load_more' => 'Load more articles',
+		'mark_all_read' => 'Mark all as read',
+		'next' => 'Next',
+		'nothing_to_load' => 'There are no more articles',
+		'previous' => 'Previous',
+	),
+	'period' => array(
+		'days' => 'days',
+		'hours' => 'hours',
+		'months' => 'months',
+		'weeks' => 'weeks',
+		'years' => 'years',
+	),
+	'share' => array(
+		'Known' => 'Known based sites',
+		'blogotext' => 'Blogotext',
+		'diaspora' => 'Diaspora*',
+		'email' => 'Email',
+		'facebook' => 'Facebook',
+		'gnusocial' => 'GNU social',
+		'jdh' => 'Journal du hacker',
+		'lemmy' => 'Lemmy',
+		'linkedin' => 'LinkedIn',
+		'mastodon' => 'Mastodon',
+		'movim' => 'Movim',
+		'pinboard' => 'Pinboard',
+		'pocket' => 'Pocket',
+		'print' => 'Print',
+		'shaarli' => 'Shaarli',
+		'twitter' => 'Twitter',
+		'wallabag' => 'wallabag v1',
+		'wallabagv2' => 'wallabag v2',
+	),
+	'short' => array(
+		'attention' => 'Warning!',
+		'blank_to_disable' => 'Leave blank to disable',
+		'by_author' => 'By:',
+		'by_default' => 'By default',
+		'damn' => 'Blast!',
+		'default_category' => 'Uncategorized',
+		'no' => 'No',
+		'not_applicable' => 'Not available',
+		'ok' => 'OK!',
```

Comment:
Like I said I'd rather see either OK or okay in both, but this is inconsistent. 
```suggestion
		'ok' => 'Okay!',
```

Reply:
I understand the consistency point here.  Will consider revising (see above discussion on okay/OK/ok)

Reply:
Favoring consistancy